custom_components/kia_uvo/services.py

```diff
@@ -49,6 +50,7 @@ async def async_handle_update(call):
 
     async def async_handle_start_climate(call):
         coordinator = _get_coordinator_from_device(hass, call)
+        climate_request_options = ClimateRequestOptions(set_temp = call.data["temperature"], duration = call.data["duration"], climate = call.data["climate"], heating = call.data["heating"])
```

Comment:
if we have any intention to default any of these inputs, we can do here as call.data.get("variable","default")